### ISSUE TITLE
Node does not automatically join cluster after restart

### ISSUE THREAD

**Issue by dkuji** [distribworks/dkron#1284]
dkron version: 3.2.1
OS: CentOS Stream8

I have configured a cluster of 3 nodes.
After rebooting the OS on node 1, dkron starts but does not automatically join the cluster.

The result of `dkron raft list-peers` after OS reboot is as follows

- node1
```
# dkron raft list-peers
Node   ID     Address      State     Voter
node1  node1  xx.xx.xx.xx  follower  true
node2  node2  xx.xx.xx.xx  follower  true
node3  node3  xx.xx.xx.xx  follower  true
```

- node2, node3
```
# dkron raft list-peers
Node   ID     Address      State     Voter
node2  node2  xx.xx.xx.xx  follower  true
node3  node3  xx.xx.xx.xx  leader    true
```

I restarted dkron on node1 and cannot join the cluster.
After restarting dkron on node1, I restarted dkron on node2 and node1 was able to join the cluster.

Is this the correct way to join a cluster?
Is it possible to get node1 to join the cluster without restarting dkron on node2 or node3?

**Comment by vcastellm** [comment]
Yes it should work as expected, one question, does node1 changed its IP address after rebooting? can you paste node1 logs?

**Comment by dkuji** [comment]
@vcastellm
Thx!
The IP address was not changed before and after the reboot.

**Comment by dkuji** [comment]
Sorry, the node is in a closed environment and I cannot post the log.

By the way, I checked the log again and found the following log output.

```
[ERR] memberlist: Conflicting address for <hostname>. Mine: <IP address>:8946 Old state: 0
[ERR] serf: Node name conflicts with another node at :::8946. Names must be unique! (Resolution enabled: true)
```
I found this issue on serf. (Consul also uses serf as well as dkron)
https://github.com/hashicorp/consul/issues/733

And in the same way, I set a 10 second sleep before starting the Dkron agent(systemd).

```
[Service]
TimeoutStartSec=100
ExecStartPre=/bin/sleep 10
```

As a result, it can now automatically join the cluster after an OS reboot.
